apps/routes-api/src/app/controllers/routes.controller.ts

```diff
@@ -10,11 +10,17 @@ class RoutesController {
     this.routesService = new RoutesService();
   }
 
-  public removeRoute = async (req: Request, res: Response, next: NextFunction) => {
+  public removeRoute = async (req: AuthenticatedRequest, res: Response, next: NextFunction) => {
     try {
-      const id = req.params.id
-      this.routesService.removeRoute(id)
-      res.sendStatus(200)
+      const id_route = req.params._id
+      const id_route_user = req.params.userId
```

Comment:
Why are you using this? You should check in the service that the route is owned by the right user. Here you're not checking this, I don't understand what this is for 🤔